jbpm-designer-backend/src/main/java/org/jbpm/designer/bpmn2/impl/Bpmn2JsonUnmarshaller.java

```diff
@@ -5990,15 +5990,8 @@ protected void applyTaskProperties(Task task,
                     Assignment a = Bpmn2Factory.eINSTANCE.createAssignment();
                     FormalExpression fromExpression = Bpmn2Factory.eINSTANCE.createFormalExpression();
                     if (assignmentParts.length > 1) {
-                        // check if custom workitem property has been updated
-                        if (isCustomElement(properties.get("tasktype"),
-                                            preProcessingData) &&
-                                (properties.get(fromPart.toLowerCase()) != null && properties.get(fromPart.toLowerCase()).length() > 0)) {
-                            fromExpression.setBody(properties.get(fromPart.toLowerCase()));
-                        } else {
-                            String replacer = decodeAssociationValue(assignmentParts[1]);
-                            fromExpression.setBody(wrapInCDATABlock(replacer));
-                        }
+                        String replacer = decodeAssociationValue(assignmentParts[1]);
```

Comment:
This breaks the update of custom elements when "customEditor" : "true" is used in the wid. unless we completely disable this feature i think we should not break it :)